src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,35 @@
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
+import PropTypes from 'prop-types';
+
+const ChatEntry = ({ id, sender, body, timeStamp, liked, onToggleLike }) => {
+  const handleLikeClick = () => {
+    onToggleLike(id);
+  };
 
-const ChatEntry = () => {
   return (
     <div className="chat-entry local">
```

Comment:
I know this was in the scaffold, but since `div` is not a semantic element, I would consider something like `article` as the outermost tag on a ChatEntry.